services/frontend/src/components/profileForms/primaryInfoForm/gedsUpdateModal/GedsUpdateModalView.jsx

```diff
@@ -145,6 +145,7 @@ const GedsUpdateModalView = ({ visibility, saveDataToDB }) => {
   const syncGedsButtonAction = async ({ paramName }) => {
     setTableLoading(true);
 
+    // eslint-disable-next-line prefer-const
     let updatedProfile = {};
```

Comment:
Why not converting this to a const? In js, a const just means you can set, in this case, updateProfile to a new value, but you can technically still update its object values

Reply:
The reason I made it let is that with const every time we change updatedProfile we are basically creating a new variable. I can change it back to const.